domains/nextor.is-a.dev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "NestorGuzman182",
+    "email": "ing.nestor.guzman@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://nestorguzman182.github.io/Portfolio/"
```

`CNAME` record must contain a domain name only, remove `https://` and `/Portfolio/`.
```suggestion
    "CNAME": "nestorguzman182.github.io"
```